macros/src/parser/mod.rs

```diff
@@ -145,6 +145,15 @@ impl ParsedStateMachine {
             // if input state is a wildcard, we need to add this transition for all states
             if transition.in_state.wildcard {
                 for (name, in_state) in &states {
+                    // skip already set input state
+                    let p = states_events_mapping
+                        .get_mut(&in_state.to_string())
+                        .unwrap();
+
+                    if p.contains_key(&transition.event.ident.to_string()) {
+                        continue;
```

Comment:
This is really nice. I've had this exact issue a few times. Thanks for the change! 

It may be worth omitting a warning to the user in this case though as there are now multiple edges defined in the state machine graph and we're electing to use the most explicit one (e.g. *State1 + Event1 = Init instead of _ + Event2 = Fault). It would be good to inform the user that we're making that decision.

Reply:
I think you shouldn't omit a warning since this is a valid syntax:

```
statemachine!{
    transitions: {
        *State1 + Event1 = State2,
        _ + Event1 = Fault,  // This is valid only after explicit definitions
    }
}
```

and here anyway we will produce an error:

```
statemachine!{
    transitions: {
        _ + Event1 = Fault,
        *State1 + Event1 = State2,
    }
}
```

I think we can produce an error here:

```
statemachine!{
    transitions: {
        *State1 + Event1 = State2,
        _ + Event1 = Fault,
        _ + Event1 = Foo, // This is unreachable
    }
}
```

what do you thing?

Reply:
Sounds appropriate to me! Valid point about explicit trumping implicit. This shouldn't be surprising and shouldn't emit an error - agreed on that front